shared/fs/header-container.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import * as Types from '../constants/types/fs'
+import {connect, type TypedState, type Dispatch} from '../util/container'
+import {navigateAppend} from '../actions/route-tree'
+
+type OwnProps = {
+  path: Types.Path,
+}
+
+const mapStateToProps = (state: TypedState, ownProps: OwnProps) => ownProps
```

Comment:
this is ok but not strictly necessary. The default mergeprops (if you don't pass one) will spread ownProps into the output automatically

Reply:
But I need to pass `mapStateToProps` to `connect`.

Reply:
not true, you can pass undefined or () => ({})